docker/sandbox/sandbox.template.yaml

```diff
@@ -166,7 +166,7 @@ metadata:
     kubernetes.io/ingress.class: traefik
 spec:
   rules:
-    - host: "${SANDBOX_NAME}.wikia.com"
+    - host: "${SANDBOX_NAME}.wikia.com", "${SANDBOX_NAME}.fandom.com"
```

Comment:
I just checked this manually, and it won't work. You need to create a separate entry.